util/base1024/decode.go

```diff
@@ -0,0 +1,35 @@
+package base1024
+
+func DecodeString(s string) ([]byte, error) {
+	//tail := false
+	//if strings.HasSuffix(s, string(TAIL)) {
+	//	s = s[0 : len(s)-len(TAIL)]
+	//	//input = input.slice(0, input.length - TAIL.length)
+	//	tail = true
+	//}
+
+	// Decode input string
+	//points := Array.from(input).map((emoji: string) => EMOJIS.indexOf(emoji))
```

Comment:
`EMOJIS` is generated in [here](https://github.com/DimensionDev/coda/blob/master/src/constants/index.ts)